src/frontend/src/flows/manage/index.ts

```diff
@@ -192,14 +192,14 @@ const displayManageTemplate = ({
         },
       })}
     </p>
+    ${pinAuthenticators.length > 0
```

Comment:
btw. would following work?

```
${pinAuthenticators.length > 0 && tempKeysSection({ authenticators: pinAuthenticators, i18n })}
```

Reply:
No, it will then print "false" into the dom, if there are no temp keys.